vars/echorouter.go

```diff
@@ -0,0 +1,26 @@
+package vars
+
```

Comment:
I am hesitant to add this change because of the enormous amount of dependencies that echo brings in.
I would prefer to see this URL Var extractor added as documentation in the Readme.